src/node_options.cc

```diff
@@ -86,6 +86,11 @@ void PerIsolateOptions::CheckOptions(std::vector<std::string>* errors) {
     return;
   }
 
+  if (per_process::cli_options->report_on_fatalerror) {
```

Comment:
many of these changes will be rendered un-necessary through https://github.com/nodejs/node/pull/32242 , but this particular change will be required, and seems sensitive to me. @addaleax - I believe you had specific concerns on accessing per-process data from per-isolate routines, in the referenced PR? can you please have a look and suggest?

Reply:
pls disregard this request, these changes are already removed, because of #32242